src/hooks/useSnackbar.tsx

```diff
@@ -49,7 +50,15 @@ export const SnackbarProvider = ({ children }: ChildrenProps) => {
         key="fta-snackbar"
       >
         <Alert onClose={handleClose} severity={alertType}>
-          {message}
+          {messageLines.length == 1
```

Comment:
This is quite poluting the code, while what you did might be reusable function. Could we put it separately in a function without needing to to do ? :
`const messageLines = message ? message.split("\n") : [""];`

Reply:
@blcham 
I solved it using `<pre>` tag instead of replacing `\n` with `<br/>`.
I guess I am an AI ...
https://dev.to/yuya0114/how-to-display-line-breaks-in-react-for-the-n-newline-character-3b0h#comment-25c4o